### PR TITLE
kollector (new formula)

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by woodruffw** [comment]
It looks like you pushed up an outdated tree; can you rebase and force-push so that only your changes are part of this PR?

**Comment by gjahesh** [comment]
I changed the Formula and did git add, commit and push --set-upstream to my branch, do I still need to rebase and force-push?

**Comment by woodruffw** [comment]
Yes, rebasing and force-pushing should fix it. I'm not sure what state your repo is in locally, but that's my best guess.

**Comment by gjahesh** [comment]
Did git rebase with but get the following error doing `git push --force` 

remote: Permission to Linuxbrew/homebrew-core.git denied to gjahesh.
fatal: unable to access 'https://github.com/Linuxbrew/homebrew-core/': The requested URL returned error: 403

`brew update` is fine but `brew doctor`  is giving 2 warnings:

```
Warning: Unbrewed .pc files were found in /usr/local/lib/pkgconfig.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected .pc files:
  /usr/local/lib/pkgconfig/python-2.7.pc

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libpython2.7.a
```
I was wondering if the warnings are the blocking the push ?
Thanks!

**Comment by woodruffw** [comment]
No, those aren't causing the error.

I think the problem is that you've opened the PR here, on Homebrew, when what you might have wanted was Linuxbrew (based on the commit history you pulled into this PR). Did you mean to submit this to us, or to Linuxbrew?

**Comment by gjahesh** [comment]
Homebrew, I did install and use linuxbrew for the PR though as you mentioned.

**Comment by woodruffw** [comment]
Yeah, that'll cause the problem. Linuxbrew's tree is out-of-sync with Homebrew's, since they need to apply Linux-specific changes to formulae.

The best way to submit a PR to Homebrew from a Linux machine is to clone homebrew-core from this organization and to create a branch within that repo (I use Linux, and that's how I develop for Homebrew).

Sorry for the confusion.

**Comment by gjahesh** [comment]
Done!Thanks!

File: Formula/kollector.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (woodruffw) [inline Formula/kollector.rb]: No need for this blank line here.

File: Formula/kollector.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+class Kollector < Formula
+  desc "Targeted de novo Gene Assembly"
+  homepage "https://github.com/bcgsc/kollector"
+  url "https://github.com/bcgsc/kollector/archive/V1.0.tar.gz"
+  version "1.0"
```
Comment (woodruffw) [inline Formula/kollector.rb]: Explicit version shouldn't be necessary, the URL is simple enough for Homebrew to pick it up.

File: Formula/kollector.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  sha256 "76dc3e03398ade0ae39b241943ce7dabcf65c09e3b30180726bfea574c462d15"
+  #tag "bioinformatics"
```
Comment (woodruffw) [inline Formula/kollector.rb]: No need for this comment.

File: Formula/kollector.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+  depends_on "abyss" => :build
+  depends_on "biobloomtools" => :build
+  depends_on "gmap-gsnap" => :build
+  depends_on "samtools" => :build
+  depends_on "bwa" => :build
+  
+  def install
+
```
Comment (woodruffw) [inline Formula/kollector.rb]: Extra blank line here.

File: Formula/kollector.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+   
+    bin.install "./bin/kollector.sh"
+    bin.install "./bin/kollector-recruit.mk"
+    bin.install "./bin/kollector-extract.sh"
+    bin.install "./bin/kollector_multiple.sh"
+    doc.install "README.md"
+    doc.install "LICENSE.txt"
+    prefix.install "test"
+    prefix.install "data"
+
```
Comment (woodruffw) [inline Formula/kollector.rb]: Extra blank line here.

**Comment by woodruffw** [comment]
Awesome, thank you. I left some minor comments on the formula.

**Comment by gjahesh** [comment]
Sorry! Don't know why Formula/ccextractor.rb is conflicting :(

**Comment by woodruffw** [comment]
It looks like you pulled it into your branch by accident. How did you amend the commit after the changes I suggested?

**Comment by woodruffw** [comment]
By the way, since this is related to bioinformatics (sorry for just realizing that now), it might be better suited to the `homebrew/science` tap.

**Comment by gjahesh** [comment]
I did rebase again and ignored to remove the commit for ccextractor.rb, then forced push, should have done commit --amend.
Should I redo every thing cloning`homebrew-science` then?

**Comment by woodruffw** [comment]
@gjahesh Yes, I think the science tap is the best place to direct this. You can ping me in on the PR there if you want help in review.

**Pull request closed without merging**